core/services/relay/evm/mercury/persistence_manager_test.go

```diff
@@ -67,7 +67,7 @@ func TestPersistenceManagerAsyncDelete(t *testing.T) {
 
 	pm.AsyncDelete(&pb.TransmitRequest{Payload: reports[0]})
 
-	time.Sleep(15 * time.Millisecond)
+	time.Sleep(20 * time.Millisecond)
```

Comment:
Should not use timing-based tests at all. Can you rewrite in a way that does not rely on `time.Sleep`

Reply:
I agree. I ended up changing the log level to debug so we can wait for it in the tests.